### DISCUSSION TITLE
Clangd doesn't give me the correct prompt on windows .

### DISCUSSION THREAD

**Discussion by jhc-hcw** [clangd/clangd#1327] [Q&A]
when I use filesystem library on windows, clangd doesn't give me the correct prompt on windows .
![image](https://user-images.githubusercontent.com/67103872/194717527-0514e1d6-d1fb-407e-b06d-6122362b056e.png)
All of about filesystem header source code are disable because it recongnised mine don't on c+17.
![image](https://user-images.githubusercontent.com/67103872/194717638-2069aeb9-1854-4d5f-a3f6-9059049406dc.png)
How can I change claned promption to c++17,or change the behavior of the library and let me use it.

**Comment by HighCommander4** [comment]
Does your project have a `compile_commands.json`? Please see https://clangd.llvm.org/installation#project-setup

**Comment by jhc-hcw** [comment]
I solved it, I just set a line in settings "clangd.fallbackFlags": ["-std=c++17"]。
![image](https://user-images.githubusercontent.com/67103872/194740370-732784ee-73b7-4254-a0cd-c55358ad9ff3.png)
